site/content/docs/5.0/components/scrollspy.md

```diff
@@ -12,11 +12,16 @@ Scrollspy has a few requirements to function properly:
 
 - It must be used on a Bootstrap [nav component]({{< docsref "/components/navs" >}}) or [list group]({{< docsref "/components/list-group" >}}).
 - Scrollspy requires `position: relative;` on the element you're spying on, usually the `<body>`.
-- When spying on elements other than the `<body>`, be sure to have a `height` set and `overflow-y: scroll;` applied.
 - Anchors (`<a>`) are required and must point to an element with that `id`.
 
 When successfully implemented, your nav or list group will update accordingly, moving the `.active` class from one item to the next based on their associated targets.
 
+{{< callout >}}
+### Spying on elements other than the `<body>`
```

Comment:
I'd say this is more a quick tip on "Scrollable containers" - not specific to spying

Reply:
My understanding is:
1. if you're not spying `body`, you'll have a scrollable container,
2. if we mention "scrollable containers",  `body` is concerned.

I'm not sure how I should reword this, I may be misunderstanding both words and concepts here.